jenkins.py

```diff
@@ -244,17 +263,15 @@ def read_config(conf):
     module_config['base_url'] = ("http://%s:%s/" %
                                  (module_config['plugin_config']['Host'], module_config['plugin_config']['Port']))
 
-    if 'ssl_certificate' in module_config['ssl_keys'] and 'ssl_keyfile' in module_config['ssl_keys']:
+    if module_config['ssl_keys']['enabled']:
```

Comment:
It seems easy to forget to turn on this flag and wonder why it is ignoring the `ssl_*` options.

Reply:
Hmm yeah I'll change it to turn on implicitly for that case since you'd never want it off then.